python/cudf/cudf/benchmarks/get_datasets.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.
+
+import os
+import shutil
+import argparse
+from collections import namedtuple
+
+# Update url and dir where datasets needs to be copied
+Dataset = namedtuple("Dataset", ["url", "dir"])
+datasets = {
+    "cuio_dataset": Dataset(
+        "https://rapidsai-data.s3.us-east-2.amazonaws.com/cudf/benchmark/"
+        "avro_json_datasets.zip",
```

Comment:
Did `black` split the string like this? This makes it really hard to correctly read as 1 string instead of 2.

Reply:
flake8 would fail

Reply:
Any other ideas on a more readable way to indicate that it's 1 string as opposed to 2?

Reply:
Something like ("blah blah" \
                          "blah blah")

Reply:
Else have to move all these options to a text file and then read it, which is not that elegant

Reply:
Using parenthesis sounds like a good compromise to show it's a single string.